### ISSUE TITLE
How to display error without "No Results" item blocking the message?

### ISSUE THREAD

**Issue by joeythomaschaske** [pozil/sfdc-ui-lookup-lwc#117]
I am setting an error on the lookup, but it is covered by the default "No Results Found" item.

How can I tell the component to stop showing the spinner and make the error message visible?

```html
<c-lookup label="Search" onsearch={handleSearch} placeholder="CA, NY, WI, etc..." errors={searchErrors}></c-lookup>
```

```javascript
  async handleSearch(event) {
        this.searchErrors = [];
        const lookup = event.target;

        try {
            const stations = await searchStations({ states: event.detail.searchTerm.split(',').map(state => (state.trim().toUpperCase())) });
            lookup.setSearchResults(stations);
        } catch (exception) {
            this.searchErrors = [
                {
                    id: 'error',
                    message: exception.body.message
                }
            ]
            lookup.setSearchResults([]);
        }
    }
```

<img width="441" alt="Screen Shot 2022-07-30 at 11 42 03 PM" src="https://user-images.githubusercontent.com/7061638/182010475-23ae307a-6000-4326-929c-3d43c1c9b6e3.png">

**Comment by joeythomaschaske** [comment]
Answer: blur the lookup element.

```javascript
  async handleSearch(event) {
        this.searchErrors = [];
        const lookup = event.target;

        try {
            const stations = await searchStations({ states: event.detail.searchTerm.split(',').map(state => (state.trim().toUpperCase())) });
            lookup.setSearchResults(stations);
        } catch (exception) {
            this.searchErrors = [
                {
                    id: 'error',
                    message: exception.body.message
                }
            ]
            lookup.setSearchResults([]);
            lookup.blur();
        }
    }
    ```

**Comment by joeythomaschaske** [comment]
Jumped the gun on this, the lookup element doesn't have a `blur` function

**Comment by pozil** [comment]
Hi @joeythomaschaske, thanks for reaching out. This is a good use case so I just released [v2.16.0](https://github.com/pozil/sfdc-ui-lookup-lwc/releases/tag/2.16.0) that addresses this.
